kano_settings/config_file.py

```diff
@@ -41,7 +41,8 @@
     'Font': 'Normal',
     'Wallpaper': 'kanux-background',
     'Parental-level': 0,
-    'Locale': 'en_US'
+    'Locale': 'en_US',
+    'display_rotate': 0
```

Comment:
On second thoughts, this should probably be removed as we're using the boot_config setters and getters rather than the ones from config_file, so this is not kept up to date. https://github.com/KanoComputing/kano-settings/blob/master/kano_settings/config_file.py#L81-L102